bin/i18n/resources/dashboard/courses/sync_out.rb

```diff
@@ -33,8 +33,6 @@ def crowdin_file_path_of(language)
           end
 
           def restore_localization(language)
-            return unless File.exist?(I18N_BACKUP_FILE_PATH)
```

Comment:
Just for clarification, could you explain why this line is removed?

Reply:
Because this condition was not in the original script. If there are no original file for a redacted file, the process should fail